cameraview/src/main/java/com/otaliastudios/cameraview/Camera1.java

```diff
@@ -250,14 +286,21 @@ void setSessionType(SessionType sessionType) {
 
     @Override
     void setLocation(Location location) {
-        Location oldLocation = mLocation;
-        mLocation = location;
-        if (isCameraAvailable()) {
-            synchronized (mLock) {
-                Camera.Parameters params = mCamera.getParameters();
-                if (mergeLocation(params, oldLocation)) mCamera.setParameters(params);
+        try {
```

Comment:
I don't think this is a right way to handle issues. App will never crash, but we would get no bug reports, or reports with incomplete stack traces.

We want this to work, if it doesn't, there's a bug in the library and we must have reports and fix it. I know that the default behavior is to throw but still.. If some component really proves to have hardware-related bugs, we can wrap that

Reply:
I guess for a shared use case you may be right. However, I think I'll keep this for my own fork. My very first priority is to prevent total crashes. At least as long as I'm still able to fetch the errors silently and thereby continue receiving crash reports.